examples/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "tinyman-js-sdk-examples",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./src/index.ts",
+  "scripts": {
+    "start:dev": "nodemon"
```

Maybe we can add start script too

```suggestion
  "scripts": {
    "start": "ts-node ./src/index.ts",
    "start:dev": "nodemon"
```

---

On second thought, we won't need nodemon anymore, it was more useful for developing examples. So I removed it, thanks!